docs/UserGuide.md

```diff
@@ -67,205 +72,219 @@ and efficient Graphical User Interface interaction.
 
-Deletes entries of warehouses or suppliers that are not needed anymore.
+Deletes a particular supplier or warehouse entry that is not needed anymore.
 
 Format: `delete ct/TYPE i/INDEX`
 
-* The TYPE specified should be one of these values: `w` / `s`.
-* The INDEX must be a positive integer, not exceeding the total length of the displayed list.
+* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
+* `INDEX` must be a positive integer, not exceeding the total length of the displayed list.
 
 Examples:
 
-* `delete ct/w i/1` Removes the warehouse at index 1 of the list of warehouses.
-* `delete ct/s i/12` Removes supplier at index 12 of the list of suppliers.
+* `delete ct/s i/12` : Removes the supplier at index 12 of the list of suppliers.
+* `delete ct/w i/1` : Removes the warehouse at index 1 of the list of warehouses.
 
-### Deletes a product in a certain warehouse or supplier : `delete`
+### Deleting a product in a supplier or warehouse : `delete`
 
-Deletes a product entry no longer stored by a certain warehouse or sold by a specific supplier.
+Deletes a product entry no longer sold by a specific supplier or stored by a specific warehouse.
 
 Format: `delete ct/TYPE i/INDEX pd/PRODUCT_NAME`
 
-* The TYPE specified should be one of these values: `pw` / `ps`.
-* The INDEX must be a positive integer, not exceeding the total length of the displayed list.
-* The PRODUCT_NAME must be an identifiable full name of the product chosen, and must starts with an alphanumeric character.
-* The product with the PRODUCT_NAME should be in the INDEX-th supplier/warehouse in the displayed list.
+* `TYPE` specified should be either `ps` for products in supplier or `pw` for products in warehouse.
+* `INDEX` must be a positive integer, not exceeding the total length of the displayed list.
+* `PRODUCT_NAME` must be an identifiable full name of the product, and it must start with an alphanumeric character.
+* `PRODUCT_NAME` is case-insensitive.
+* `PRODUCT_NAME` must exist in the current supplier or warehouse before it can be deleted.
 
 Examples:
 
-* `delete ct/pw i/1 pd/Panadol` Removes product with the name _Panadol_ from
-the warehouse at index 1 of the list of warehouses.
-* `delete ct/ps i/12 pd/Aspirin` Removes product with the name _Aspirin_
-from the supplier at index 12 of the list of suppliers.
+* `delete ct/pw i/1 pd/Panadol` : Removes the `Panadol` product from the warehouse at index 1 of the list of warehouses.
+* `delete ct/ps i/12 pd/Aspirin` : Removes the `Aspirin` product from the supplier at index 12 of the list of suppliers.
 
 <div markdown="span" class="alert alert-info">
-**:information_source: Note:** If additional PRODUCT_NAME behind prefix `pd/` is added when the TYPE `ct/` given is "w" or "s", the PRODUCT_NAME will be ignored
-as the entire warehouse/supplier entry is removed.
+**:information_source: Note:** The prefix `pd/` will be ignored when `ct/` given is `s` or `w`.
```

Comment:
Somehow **:information_source: Note:** is still not showing correctly if I try to view this md file here. 🤔

Reply:
Hmm it didn't show up previously in IntelliJ as well! I suspect it will only show when we view it on GH!